src/postcss.js

```diff
@@ -98,6 +99,13 @@ export default postcss.plugin('postcss-preset-env', opts => {
 	);
 
 	return (root, result) => {
+		const majorVersion = parseInt(result.processor.version.split('.')[0]);
+		if (majorVersion > 7) {
+			console.log('');
+			console.log(boxen(`This version of postcss-preset-env is not optimised to work with PostCSS 8.\nPlease update to version ${majorVersion} of PostCSS Preset Env.\n\n If you find issues, you can report it at: \nhttps://github.com/csstools/postcss-plugins/issues/new/choose`, { padding: 1, textAlignment: 'center' }));
+			console.log('');
+		}
```

Comment:
Can we take the output from `boxen` once and use that hard coded?

`boxen` has too many `sindresorhus` authored micro packages in it's chain.

Reply:
Great suggestion! Fixed